### PR TITLE
Update COMMUNITY-PROJECTS.md - ipfsUploader

### PR DESCRIPTION
**Pull request by rej156**

### CODE REVIEW

**Comment by michaelsena** [comment]
@rej156 thanks for submitting this awesome project and PR. Would you mind also adding your keys to [this list](https://github.com/uport-project/3box-js/blob/develop/KEY-USAGE.md#private-data)? After you do that I'll merge them in! 👌

File: COMMUNITY-PROJECTS.md [hunk 1]
```diff
@@ -28,3 +28,9 @@ Ongoing
 
 #### ipfsUploader
 Ongoing
```
Comment (michaelsena) [inline COMMUNITY-PROJECTS.md]: Would you mind if we linked to your repo here? > https://github.com/rej156/ipfs-uploader <

**Comment by rej156** [comment]
Done and done
@michaelsena 
https://github.com/uport-project/3box-js/pull/103

**Comment by michaelsena** [comment]
Merged @rej156 :+1:

**Pull request merged**